receipt/SaleReceipt_fr_CA-QC.tpl

```diff
@@ -1029,7 +1037,7 @@ table.payments td.label {
 								<!-- Gift Card -->
 								{% if Payment.amount > 0 %}
 									<tr>
-										<td class="label">Prélèvement sur carte cadeau</td>
+										<td class="label">Prélèvement sur carte cadeau {{Payment.CreditAccount.code}}</td>
```

Comment:
**Nit**: `{Payment.CreditAccount.code}}` is added here, but not on the other language files. It appears to be present on the related files in the private repo. Should this be added in this PR as well?

Reply:
Nice catch! **Addressed**